src/erf.jl

```diff
@@ -4,7 +4,7 @@ using Base.Math: @horner, libm
 using Base.MPFR: ROUNDING_MODE
 
 for f in (:erf, :erfc)
-    @eval begin
+    @eval beginhttps://github.com/discourse/discourse/tree/master/config/locales
```

Comment:
whoops?

Reply:
Seems like valid syntax to me :trollface:

Reply:
Sorry. For few months, my linux laptop is pasting unexpectedly the buffer from time to time. I've just made some experiments now. Touchpad tapping is disable for years, but right top angle touchpad tapping is strangely enabled now. I will remove that.